core/src/main/java/com/sequenceiq/cloudbreak/service/rdsconfig/RedbeamsDbServerConfigurer.java

```diff
@@ -0,0 +1,84 @@
+package com.sequenceiq.cloudbreak.service.rdsconfig;
+
+import static com.sequenceiq.cloudbreak.api.endpoint.v4.common.DatabaseVendor.POSTGRES;
+
+import java.util.Collections;
+import java.util.Date;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.inject.Inject;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.common.ResourceStatus;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.database.base.DatabaseType;
+import com.sequenceiq.cloudbreak.domain.RDSConfig;
+import com.sequenceiq.cloudbreak.domain.stack.Stack;
+import com.sequenceiq.cloudbreak.domain.stack.cluster.Cluster;
+import com.sequenceiq.cloudbreak.exception.NotFoundException;
+import com.sequenceiq.cloudbreak.service.secret.service.SecretService;
+import com.sequenceiq.redbeams.api.endpoint.v4.databaseserver.responses.DatabaseServerV4Response;
+
+@Service
+public class RedbeamsDbServerConfigurer {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(RedbeamsDbServerConfigurer.class);
+
+    private static final String JDBC_PATTERN = "\\/\\/(.*?):(\\d*)";
```

Comment:
The DatabaseCommon class has some powerful methods for parsing JDBC URLs that can be used instead.